front/cypress/e2e/survey_builder/create_and_fill_survey.cy.ts

```diff
@@ -73,7 +82,7 @@ describe.skip('Survey builder', () => {
     cy.contains(questionTitle).should('exist');
 
     // Try going to the next page without filling in the survey
-    cy.get('[data-cy="e2e-next-page"]').should('be.visible').click();
+    cy.get('[data-cy="e2e-submit-form"]').should('be.visible').click();
```

Comment:
The next page button now already becomes the submit form button, if the frontend knows that the next page will be the end page.